pkg/features/kube_features.go

```diff
@@ -708,6 +708,12 @@ const (
 	//
 	// Enable Scope and Namespace fields on IngressClassParametersReference.
 	IngressClassNamespacedParams featuregate.Feature = "IngressClassNamespacedParams"
+
+	// owner: @jayunit100 @abhiraut
```

Comment:
This should also say include @rikatz ;)

Reply:
Added while needed to rebase kube_features (again!)

Can I have a lgtm here? :P